tee-worker/ts-tests/integration-tests/tsconfig.json

```diff
@@ -14,6 +14,7 @@
     "ts-node": {
         "esm": true,
         "experimentalResolver": true,
-        "experimentalSpecifierResolution": "node"
+        "experimentalSpecifierResolution": "node",
+        "transpileOnly": true
```

Comment:
Adding this to mitigate the `error:NULL` messages from mocha; see [this issue](https://github.com/mochajs/mocha/issues/5085) /cc @jonalvarezz @0xverin